greengrassTestSuite.py

```diff
@@ -349,7 +419,8 @@ def get_ggcore_device_status(self, timeout, thing_group_name):
 
         while timeout > 0:
             return_val = self._ggClient.get_core_device(
-                coreDeviceThingName=things_in_group["things"][0])
+                coreDeviceThingName=things_in_group["things"][0]
+            )
 
             if return_val == None or return_val["status"] != "HEALTHY":
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The `==` and `!=` operators use the compared objects&#39; ` __eq__` method to test if they are equal. To check if an object is a singleton, such as `None`, we recommend that you use the `is` identity comparison operator.

[Learn more](https://docs.python.org/3/reference/expressions.html#is)